index.js

```diff
@@ -1,31 +1,44 @@
+const dotProp = require('dot-prop');
+const readPkgUp = require('read-pkg-up');
+
 const eslint = require('./src/eslint');
 const eslintComments = require('./src/eslint-comments');
 const getOffMyLawn = require('./src/get-off-my-lawn');
 const imprt = require('./src/import');
 const jsxA11y = require('./src/jsx-a11y');
-const jest = require('./src/jest');
+const jestRules = require('./src/jest');
 const node = require('./src/node');
 const objects = require('./src/objects');
-const preferObjectSpread = require('./src/prefer-object-spread');
 const react = require('./src/react');
+const reactHooks = require('./src/react-hooks');
+const reactNative = require('./src/react-native');
 const security = require('./src/security');
 const unicorn = require('./src/unicorn');
 
+const getReactVesion = () => {
```

Comment:
Is this used to determine whether to enforce hooks lints?

Reply:
No. [eslint-plugin-react recently added a `version` property](https://github.com/yannickcr/eslint-plugin-react#configuration). When it's not supplied, it defaults to `detect`, but if you use `get-off-my-lawn` without using React (like a pure node module, for example) and it cannot detect a version of React, then it throws warnings when running lint. This circumvents the warning by explicitly setting it, though I have a different idea on how to solve this problem now.